airflow/www/templates/airflow/dags.html

```diff
@@ -313,6 +316,12 @@ <h2>DAGs</h2>
       }
     });
 
+    $('#search_form').on('reset', function() {
+      var query = new URLSearchParams(window.location.search);
+      query.delete('search');
```

Comment:
Should we remove page no also? I think the user should always be redirected to the first page.

Reply:
Ah, yes, good call!

Reply:
Oh yes, good catch.

Reply:
Instead of deleting the `page` param, I had to set it to `0`. Because the page value is maintained in the session cookie, that value was re-added to to the URL after deletion. By setting to `0`, that will update the cookie value as well.

Reply:
I take that back… I was confusing the page param with how the search/tags/status params were being handled. Updated to delete the page param completely.